sos/report/__init__.py

```diff
@@ -506,6 +507,25 @@ def _get_block_devs(self):
             self.soslog.error("Could not get block device list: %s" % err)
             return []
 
+    def _get_tape_devs(self):
+        """Enumerate a list of tape devices on this system so that plugins
+        can iterate over them
+
+        These devices are used by add_device_cmd() in the Plugin class.
+        """
+        try:
+            devs = []
+            devdirs = [
+                'scsi_tape',
+                'lin_tape'
```

Comment:
minor nit, comma at the end. But all good all round

Reply:
I will merge the PR and add the coma to it.